sql/parse/parse.go

```diff
@@ -1384,11 +1386,16 @@ func columnOrderToColumnOrder(order *sqlparser.ColumnOrder) *sql.ColumnOrder {
 }
 
 func convertDropTable(ctx *sql.Context, c *sqlparser.DDL) (sql.Node, error) {
-	tableNames := make([]string, len(c.FromTables))
+	dropTables := make([]sql.Node, len(c.FromTables))
+	dbName := c.FromTables[0].Qualifier.String()
```

Comment:
As far as I can tell, there's no reason to enforce this. The resolve_tables code should work for any number of DBs.